### PR TITLE
fix: always include timestamp & coords on observation position

### PR DESCRIPTION
**Pull request by EvanHahn**
We should treat these values as required.

### CODE REVIEW

File: src/frontend/screens/ObservationCreate/useMostAccurateLocationForObservation.ts [hunk 1]
```diff
@@ -45,17 +45,14 @@ export function useMostAccurateLocationForObservation() {
         },
         debounceLocation()(location => {
           if (ignore) return;
-
-          const position: Position = {mocked: false};
```
Comment (achou11) [inline src/frontend/screens/ObservationCreate/useMostAccurateLocationForObservation.ts:49]: the `Position` type is now an unused import in this file. mind removing it?

File: src/frontend/screens/Settings/CreateTestData.tsx [hunk 2]
```diff
@@ -258,7 +258,7 @@ function useCreateFakeObservationsMutation() {
       const promises = [];
 
       for (let i = 0; i < count; i++) {
-        const fakeCoordinates = randomPosition({
+        const [lon = 0, lat = 0] = randomPosition({
```
Comment (achou11) [inline src/frontend/screens/Settings/CreateTestData.tsx]: nit: would probably prefer to just throw if `randomPosition()` doesn't return a tuple of 2 elements here. 0 is unfortunately a [significant value](https://en.wikipedia.org/wiki/Null_Island) in the context of geospatial coordinates.

the TS typing they have (`number[]`) is inaccurate, as the implementation always returns a tuple:

https://github.com/Turfjs/turf/blob/6889237200a088fa5539f92f361fc96135e09ac1/packages/turf-random/index.ts#L30-L44

**Review by achou11 — DISMISSED** [review]
lgtm!

File: src/frontend/screens/ManualGpsScreen/index.tsx
Comment (achou11) [inline src/frontend/screens/ManualGpsScreen/index.tsx:1]: same comment about removing the `Position` type import in this file

File: src/frontend/screens/Settings/CreateTestData.tsx (hunk 2 above)
Comment (EvanHahn) [reply src/frontend/screens/Settings/CreateTestData.tsx]: Fixed in 9142abf01fbfad057de4c22a46e03fc49b4cb586. Also opened https://github.com/DefinitelyTyped/DefinitelyTyped/pull/70521 to fix the upstream types.

File: src/frontend/screens/ManualGpsScreen/index.tsx
Comment (EvanHahn) [reply src/frontend/screens/ManualGpsScreen/index.tsx:1]: Removed in 2128be9777a12e986f749e0993e3dcc74db8096f.

**Comment by EvanHahn** [comment]
Closing, because I think we plan to do this in a pretty different way.

**Pull request closed without merging**